201-ultra-managed-disk/azuredeploy.json

```diff
@@ -0,0 +1,72 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "managedDiskName": {
+        "type": "string",
+        "metadata": {
+          "description": "Name of the managed disk to be copied"
+        }
+      },
+      "logicalSectorSize": {
+        "type": "int",
+        "defaultValue": 4096,
+        "allowedValues": [
+            512,
+            4096
+        ],        
+        "metadata": {
+          "description": "Logical Sector Size. Recommended 4096 unless application requires 512 bytes sector size support"
+        }
+      },
+      "diskSizeinGiB": {
+        "type": "int",
+        "defaultValue": 8,
```

Comment:
are there min/max or allowedValues on these next few params?

Reply:
Address this in latest change